noxfile.py

```diff
@@ -448,7 +455,9 @@ def check_changelog(session: nox.Session):
 
     if not fragments:
         session.error(
-            "Changelog entry not found, please add one (or more) to `newsfragments` directory. For more information see https://github.com/PyO3/pyo3/blob/main/Contributing.md#documenting-changes"
+            "Changelog entry not found, please add one (or more) to the `newsfragments` directory.\n"
+            "Alternatively, start the PR title with `docs:` if this PR is a docs-only PR.\n"
+            "See https://github.com/PyO3/pyo3/blob/main/Contributing.md#documenting-changes for more information."
```

Comment:
I reordered the last sentence because nox was appending a final full stop to the URL which was breaking the link in Github Actions (would send to the wrong anchor).